diff -Nupr src.orig/drivers/tty/tty_buffer.c src/drivers/tty/tty_buffer.c
--- src.orig/drivers/tty/tty_buffer.c	2024-01-22 16:10:11.329974044 -0500
+++ src/drivers/tty/tty_buffer.c	2024-01-22 16:11:34.268100860 -0500
@@ -256,6 +256,9 @@ static int __tty_buffer_request_room(str
 	struct tty_buffer *b, *n;
 	int left, change;
 
+	if (!size)
+		printk("kpatch-test: testing .smp_locks section changes\n");
+
 	b = buf->tail;
 	if (b->flags & TTYB_NORMAL)
 		left = 2 * b->size - b->used;
