diff -Nupr src.orig/kernel/fork.c src/kernel/fork.c
--- src.orig/kernel/fork.c	2025-07-09 12:33:49.246210770 -0400
+++ src/kernel/fork.c	2025-07-09 12:34:01.721210619 -0400
@@ -1837,10 +1837,18 @@ static void posix_cpu_timers_init_group(
 	posix_cputimers_group_init(pct, cpu_limit);
 }
 
+static void kpatch_foo(void)
+{
+	if (!jiffies)
+		printk("kpatch copy signal\n");
+}
+
 static int copy_signal(unsigned long clone_flags, struct task_struct *tsk)
 {
 	struct signal_struct *sig;
 
+	kpatch_foo();
+
 	if (clone_flags & CLONE_THREAD)
 		return 0;
 
