diff -Nupr src.orig/drivers/tty/tty_buffer.c src/drivers/tty/tty_buffer.c
--- src.orig/drivers/tty/tty_buffer.c	2021-08-02 14:27:21.693019982 -0400
+++ src/drivers/tty/tty_buffer.c	2021-08-02 14:28:26.432743695 -0400
@@ -263,6 +263,9 @@ static int __tty_buffer_request_room(str
 	struct tty_buffer *b, *n;
 	int left, change;
 
+	if (!size)
+		printk("kpatch-test: testing .smp_locks section changes\n");
+
 	b = buf->tail;
 	if (b->flags & TTYB_NORMAL)
 		left = 2 * b->size - b->used;
