diff -Nupr src.orig/fs/readdir.c src/fs/readdir.c
--- src.orig/fs/readdir.c	2021-09-10 14:03:43.213999252 -0400
+++ src/fs/readdir.c	2021-09-10 14:03:55.853056254 -0400
@@ -245,6 +245,7 @@ static int filldir(struct dir_context *c
 	if (prev_reclen && signal_pending(current))
 		return -EINTR;
 	dirent = buf->current_dir;
+	asm("nop");
 	prev = (void __user *) dirent - prev_reclen;
 	if (!user_write_access_begin(prev, reclen + prev_reclen))
 		goto efault;
