diff -Nupr src.orig/kernel/fork.c src/kernel/fork.c
--- src.orig/kernel/fork.c	2021-09-10 14:03:43.552000776 -0400
+++ src/kernel/fork.c	2021-09-10 14:04:49.677299005 -0400
@@ -1560,10 +1560,18 @@ static void posix_cpu_timers_init_group(
 	posix_cputimers_group_init(pct, cpu_limit);
 }
 
+void kpatch_foo(void)
+{
+	if (!jiffies)
+		printk("kpatch copy signal\n");
+}
+
 static int copy_signal(unsigned long clone_flags, struct task_struct *tsk)
 {
 	struct signal_struct *sig;
 
+	kpatch_foo();
+
 	if (clone_flags & CLONE_THREAD)
 		return 0;
 
